cmd/argocd-repo-server/commands/argocd_repo_server.go

```diff
@@ -207,6 +209,7 @@ func NewCommand() *cobra.Command {
 	command.Flags().BoolVar(&allowOutOfBoundsSymlinks, "allow-oob-symlinks", env.ParseBoolFromEnv("ARGOCD_REPO_SERVER_ALLOW_OUT_OF_BOUNDS_SYMLINKS", false), "Allow out-of-bounds symlinks in repositories (not recommended)")
 	command.Flags().StringVar(&streamedManifestMaxTarSize, "streamed-manifest-max-tar-size", env.StringFromEnv("ARGOCD_REPO_SERVER_STREAMED_MANIFEST_MAX_TAR_SIZE", "100M"), "Maximum size of streamed manifest archives")
 	command.Flags().StringVar(&streamedManifestMaxExtractedSize, "streamed-manifest-max-extracted-size", env.StringFromEnv("ARGOCD_REPO_SERVER_STREAMED_MANIFEST_MAX_EXTRACTED_SIZE", "1G"), "Maximum size of streamed manifest archives when extracted")
+	command.Flags().BoolVar(&managedByArgo, "managed-by-argo", env.ParseBoolFromEnv("ARGOCD_REPO_SERVER_MANAGED_BY_ARGO", false), "Set managed-by label for every k8s resource generated by argo Current Status: Alpha (Since v2.8.0)")
```

Comment:
Is it true that this is considered Alpha since `v2.8.0`?

Reply:
hmm, I'm guessing this was originally thought by the author that it would get out in that release. What is the protocol for alpha features? Do I just update this with the next slated release once everything else has been approved? `v2.11.0` as of now?

Reply:
I have no idea. We'll have to wait for @blakepettersson or some other maintainer to get back to us on that.

Reply:
If at all, this should just denote "Alpha", without  any version information.

Then, this is a pretty small, non-intrusive feature. I don't believe it should be marked alpha at all.